heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -38,6 +41,7 @@ require "cjson"
 local fx = require "fx"
 require "string"
 
+local duplicate_original = read_config("duplicate_original")
```

Comment:
Add a comment describing that this setting controls whether or not we re-inject the raw message.